src/types/libdef/npm/query-string_v6.x.x.js

```diff
@@ -0,0 +1,36 @@
+// flow-typed signature: 9096718d5c8abc7d9ae9aadbe0ce565a
+// flow-typed version: 6d8676cf5a/query-string_v6.x.x/flow_>=v0.32.x
+
+declare module 'query-string' {
+  declare type ArrayFormat = 'none' | 'bracket' | 'index';
+  declare type ParseOptions = {|
+    arrayFormat?: ArrayFormat,
+  |};
+
+  declare type StringifyOptions = {|
+    arrayFormat?: ArrayFormat,
+    encode?: boolean,
+    strict?: boolean,
+    sort?: false | (<A, B>(A, B) => number),
+  |};
+
+  declare type ObjectParameter = string | number | boolean | null | void;
+
+  declare type ObjectParameters = $ReadOnly<{
```

Comment:
This is something I added because there was an issue with our code, but I also started a PR for upstream: https://github.com/flow-typed/flow-typed/pull/3193 so I think it's OK.